packages/es-dev-server/src/plugins/browserReloadPlugin.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+import { Plugin } from '../Plugin';
+import debounce from 'debounce';
+
+export interface BrowserReloadPluginConfig {
+  watchDebounce: number;
+}
+
+export function browserReloadPlugin(cfg: BrowserReloadPluginConfig): Plugin {
+  return {
+    serverStart({ fileWatcher, messageChannel }) {
+      if (messageChannel) {
+        function onFileChanged() {
+          console.log('Reloading connected browsers...');
+          console.log('');
+          messageChannel?.sendMessage({ name: 'reload' });
```

Comment:
```suggestion
          messageChannel?.sendMessage?.({ name: 'reload' });
```

Reply:
If the `messageChannel` is defined, it surely has a `sendMessage` function since that's guaranteed by types.